docs/integration/KafkaIntegration.md

```diff
@@ -2,12 +2,12 @@
 sidebar_position: 1
 ---
 
-# Kafka
+# Kafka | Streaming only
 
 ## Prerequisites
 
 To fully understand how Nussknacker works with Kafka topics, it's best to read the following first:
-* [Role of Schema Registry](/about/TypicalImplementation)
+* [Role of Schema Registry](/about/TypicalImplementation/)
```

Comment:
should be https://docs.nussknacker.io/documentation/about/TypicalImplementationStreaming

Reply:
updated